WcaOnRails/spec/factories/competitions.rb

```diff
@@ -76,5 +76,11 @@
       with_delegate
       showAtAll true
     end
+
+    trait :entry_fee do
+      base_entry_fee_lowest_denomination 1000
+      currency_code "AUD"
+      connected_stripe_account_id "acct_19ZQVmE2qoiROdto"
```

Comment:
Your call if you think it's valuable, but I might be inclined to comment on where this id came from, and how to change/recreate it. I'm probably going to forget that we actually have 2 stripe accounts.

A side note: what will happen in production if someone connects a Stripe account in in testing mode? Just their mistake, and people will be able to register without actually paying anything?